code/frontend/MakeItSo/MakeItSo/MakeItSo.docc/Tutorials/Building a simple todo list UI.tutorial

```diff
@@ -0,0 +1,473 @@
+        
+        @Code(name: "Reminder.swift", file: building-todo-ui-reminder-02-reminder-02.swift, previousFile: building-todo-ui-reminder-02-reminder-01.swift)
+      }
+      
+      @Step {
+        Navigate back to `ContentView.swift`. The compile error should now be resolved, and the error message will disappear. 
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-02-contentview-07.swift) {
+          @Image(source: building-todo-ui-reminder-02-contentview-07.png, alt: "")
+        }
+      }
+      
+      The `List` view iterates over all elements of the collection you pass in as the first parameter - in this case, `reminders`. Inside the `List` view's closure, you can access the individual elements of the collection using the closure parameter, `reminder`.
+      
+      @Step {
+        Update the views inside the `List` view to display the `title` and `completion` state of the current collection element.  
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-02-contentview-08.swift) {
+          @Image(source: building-todo-ui-reminder-02-contentview-08.png, alt: "")
+        }
+      }
+      
+    }
+  }
+  
+  @Section(title: "Mark Reminders as Completed") {
+    @ContentAndMedia {
+      The main purpose of a todo list app is to allow users to get things done. In this step, you will turn the static `Image` view into an interactive component that allows users to mark tasks as `completed`.
+      
+      @Image(source: placeholder-image.png, alt: "")
+    }
+    
+    @Steps {
+      @Step {
+        In the _Project_ navigator, select `ContentView.swift`.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-03-contentview-01.swift, previousFile: building-todo-ui-reminder-03-contentview-00.swift) {
+          @Image(source: building-todo-ui-reminder-03-contentview-01.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Use the `onTapGesture` view modifier to register an event handler that gets called when the user taps on the `Image` view.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-03-contentview-02.swift)
+      }
+      
+      @Step {
+        Toogle the `completion` status of the respective reminder.
+        
+        > Warning: This will result in a compiler error saying that you _cannot use mutating member on immutable value: 'reminder' is a 'let' constant._. You will fix this in the next step.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-03-contentview-03.swift) {
+          @Image(source: building-todo-ui-reminder-03-contentview-03.png, alt: "")
+        }
+      }
+      
+      Explain this briefly. You can learn more about `List` bindings in this blog post: [SwiftUI List Bindings - Behind the Scenes](https://peterfriese.dev/posts/swiftui-list-item-bindings-behind-the-scenes/)
```

Comment:
Is this "Explain this briefly" supposed to be here?

Reply:
Thanks for catching this!